datahub-classify/tests/test_predictions.py

```diff
@@ -11,42 +11,130 @@
 
 from datahub_classify.helper_classes import ColumnInfo, Metadata
 from datahub_classify.infotype_predictor import predict_infotypes
-from datahub_classify.sample_input import input1 as input_dict
-from datahub_classify.supported_infotypes import infotypes_to_use
+from datahub_classify.reference_input import input1 as input_dict
```

Comment:
It looks like this reference input is only used in tests. As such, that file should live in the tests directory instead of src

Reply:
This file is the default input configuration. Whenever the user/ product team will call the API, this default config will be used with/ without modification as per requirement. So this file was kept in src instead of tests.

Reply:
Right, that is hardcoded in datahub_classifier config at this point, however I plan to import from this file going forward, so that, when new info types are added in datahub-classify, they are supported right away.

Reply:
got it, sounds good